data/abilities.js

```diff
@@ -1089,6 +1093,7 @@ exports.BattleAbilities = {
 		onResidual: function (pokemon) {
 			if (pokemon.status && this.isWeather(['raindance', 'primordialsea'])) {
 				this.debug('hydration');
+				this.add('-activate', pokemon, 'ability: Hydration');
```

Comment:
Officially, if an ability activation is a standalone message, protocol should use an `-ability` tag. But realistically, it should be something like `-curestatus|POKEMON|...|[from] ability: Hydration` if possible. I just don't know if it is.